app.rb

```diff
@@ -0,0 +1,158 @@
+require './model/book'
+require './model/game'
+require './model/music_album'
+require './model/genre'
+require './model/author'
+require './model/label'
+require './data'
+
+class App
+  def initialize
+    @data = Data.new
+    @data.load_authors
+    @data.load_games
+    @data.load_albums
+    @data.load_genres
+    @data.load_books
+    @data.load_labels
+  end
+
+  def inputs(show)
+    outputs = []
+    show.each do |s|
+      print s
+      outputs.push(gets.chomp)
+    end
+    outputs
+  end
+
+  def add_author
+    puts 'Enter first name'
+    first_name = gets.chomp
+    puts 'Enter last name'
+    last_name = gets.chomp
+    new_author = Author.new(nil, first_name, last_name)
+    @data.add_author(new_author)
+    @data.authors << new_author
+    new_author
+  end
+
+  def add_game
+    puts 'Is it multiplayer [Y / N]'
+    multiplayer = gets.chomp.downcase == 'y'
+    puts 'Enter last played at:'
+    last_played_at = gets.chomp
+    puts 'Enter the publish date in:'
+    publish_date = gets.chomp
+    new_game = Game.new(nil, publish_date, multiplayer, last_played_at)
+    puts "Enter author details\n"
+    new_author = add_author
+    new_game.add_author(new_author)
+    @data.add_game(new_game)
+    @data.games << new_game
+    puts 'Game created successfully'
+  end
+
+  def add_book
+    puts 'Enter publisher'
+    publisher = gets.chomp
+    puts 'Enter publish date'
+    publish_date = gets.chomp
+    puts 'Enter the cover state'
+    cover_state = gets.chomp
+    new_book = Book.new(nil, publish_date, publisher, cover_state)
+    puts "Enter label details\n"
+    new_label = add_label
+    new_book.add_label(new_label)
+    @data.add_book(new_book)
+    @data.books << new_book
+    puts 'Book created successfully'
+  end
+
+  def add_label
+    puts 'Enter title'
+    title = gets.chomp
+    puts 'Enter color'
+    color = gets.chomp
+    new_label = Label.new(nil, title, color)
+    @data.add_label(new_label)
+    @data.labels << new_label
+    new_label
+  end
```

Comment:
- [ ] I got the console error on the screenshot below when trying to add a book:

![Screenshot from 2022-11-24 23-12-10](https://user-images.githubusercontent.com/63087491/203869319-5ad4cb4a-3a7f-4d50-9ace-9da865fa4795.png)

Let's kindly fix it.

Reply:
## Required
- [ ] I am still getting the same error please address it
<img width="793" alt="Screen Shot 2022-11-25 at 11 22 47" src="https://user-images.githubusercontent.com/44745855/203945860-6713c810-1845-475d-9d32-d668bc9d773e.png">